DevCycle/DVCClient.swift

```diff
@@ -104,15 +104,26 @@ public class DVCClient {
             
             if let sseURL = self.config?.userConfig?.sse?.url {
```

Comment:
can we rip out sse code here so it's not a mess of nested ifs, and use guard statements to flatten the logic?

Reply:
yeah!